src/søknad/steg/5-aktivitet/aksjeselskap/Aksjeselskap.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { FC, useEffect, useState } from 'react';
+import TittelOgSlettKnapp from '../../../../components/TittelOgSlettKnapp';
+import { Undertittel } from 'nav-frontend-typografi';
+import SlettKnapp from '../../../../components/knapper/SlettKnapp';
+import classnames from 'classnames';
+import FeltGruppe from '../../../../components/gruppe/FeltGruppe';
+import { Input } from 'nav-frontend-skjema';
+import InputLabelGruppe from '../../../../components/gruppe/InputLabelGruppe';
+import { hentTittelMedNr } from '../../../../language/utils';
+import { useIntl } from 'react-intl';
+import { hentTekst } from '../../../../utils/søknad';
+import {
+  EAksjeselskap,
+  IAksjeselskap,
+} from '../../../../models/steg/aktivitet/aktivitet';
+
+interface Props {
+  egetAS: IAksjeselskap[];
+  settEgetAS: (egetAS: IAksjeselskap[]) => void;
+  aksjeselskapnummer: number;
+}
+
+const Aksjeselskap: FC<Props> = ({
+  egetAS,
+  settEgetAS,
+  aksjeselskapnummer,
+}) => {
+  const intl = useIntl();
+  const aksjeselskapFraSøknad = egetAS?.find((aksjeselskap, index) => {
+    if (index === aksjeselskapnummer) return aksjeselskap;
+  });
+  const [aksjeselskap, settAksjeselskap] = useState<IAksjeselskap>(
+    aksjeselskapFraSøknad!
+  );
+
+  useEffect(() => {
+    const endretAksjeselskap = egetAS?.map((aksjeselskapFraSøknad, index) => {
+      if (index === aksjeselskapnummer) return aksjeselskap;
+      else return aksjeselskapFraSøknad;
+    });
+    settEgetAS(endretAksjeselskap);
+    // eslint-disable-next-line
+  }, [aksjeselskap]);
+
+  const aksjeselskapTittel = hentTittelMedNr(
+    egetAS!,
+    aksjeselskapnummer,
+    intl.formatMessage({ id: 'egetAS.label.aksjeselskap' })
+  );
+  const navnLabel: string = hentTekst('egetAS.label.navn', intl);
+  const arbeidsmengdeLabel: string = hentTekst(
+    'arbeidsforhold.label.arbeidsmengde',
+    intl
+  );
+
+  const fjernAksjeselskap = () => {
+    if (egetAS && egetAS.length > 1) {
```

Comment:
Bare for å dobbeltsjekke; man skal ikke kunne fjerne alle AS-ene ?

Reply:
Korrekt. I følge skissene så skal man alltid ha en bolk med felter for aksjeselskap. Fjern knappen skal forsvinne/ikke fungere dersom man ikke har flere enn 1 aksjeselskap igjen i lista 😊 

![Skjermbilde 2020-06-05 kl  09 37 34](https://user-images.githubusercontent.com/8249453/83849789-3b094500-a710-11ea-9315-6f239a7bd611.png)

![Skjermbilde 2020-06-05 kl  09 37 40](https://user-images.githubusercontent.com/8249453/83849795-3d6b9f00-a710-11ea-98a3-d0b2bfc4f659.png)